portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -559,6 +559,92 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test asserts that user can filter app by Category in marketplace store."
+	@priority = "3"
+	test CanFilterMarketplaceStoreAppsByCategories {
```

Comment:
If the _filterby_ name is Category, please also using Category in test name, otherwise, changing the test and task description accordingly. Thanks.